### ISSUE TITLE
[QUESTION] Loss increase when training DINO

### ISSUE THREAD

**Issue by YingdiZhang** [lightly-ai/lightly-train#477]
### 🤔 What’s your question?

Thanks for the great job，When training SSL model with DINO，I found the loss curve decrease first and increase later. How to solve this problem?

### 🧠 What did you try?

Explain what you’ve already looked into or attempted.

### 📌 Related code or context (if any)
import lightly_train
from ultralytics import YOLO

if __name__ == "__main__":
    model = YOLO("yolov5s.yaml")

    lightly_train.train(
        out= "yolo-s_ssl/epch300gpu4btch1024", 
        data="/data/data_zyd/foundation_model_data",
        model=model,
        method="dino", 
        accelerator="gpu",
        devices=[4,5,6,7],# [0,1,2,4,5,6,7,3],
        epochs=300, 
        batch_size=1024, 
}
    )

```python
# Related snippet if relevant
```

**Comment by stegmuel** [comment]
Hi,
This behavior is most likely expected and not problematic.
I shared some DINOv2 loss curves in #235.
Does your `total_loss` curve look similar?

**Comment by YingdiZhang** [comment]
Thanks for your reply. This is the loss curve, it increase a lot after some iteration and dosen't seem normal.

<img width="1400" height="1052" alt="Image" src="https://github.com/user-attachments/assets/f88cbe97-74c8-416b-b81a-dbbfe2dbee42" />

**Comment by stegmuel** [comment]
It looks relatively normal to me. 
There are a lot of dynamic behavior throughout training (weight decay, temperature, learning rate) and in the self-supervised learning paradigm smaller losses are not that correlated with better models.

**Comment by YingdiZhang** [comment]
Thanks for your reply，it helps
